key.go

```diff
@@ -469,18 +468,12 @@ var sequences = map[string]Key{
 	"\x1b\x1b[32~": {Type: KeyF18, Alt: true},
 	"\x1b\x1b[33~": {Type: KeyF19, Alt: true},
 	"\x1b\x1b[34~": {Type: KeyF20, Alt: true},
-}
-
-// Hex code mappings.
-var hexes = map[string]Key{
-	"1b0d": {Type: KeyEnter, Alt: true},
-	"1b7f": {Type: KeyBackspace, Alt: true},
```

Comment:
Just checking in if this was removed on purpose? Since you added a test for the two sequences above, I assume it's working fine tho.

Reply:
Yes as the commit message says this is now unneeded. It became obsolete when we fixed the alt key a few PRs ago.

Reply:
I’ll add that I’m thrilled to see those gone. Thanks, @knz.